test/transferSpec.js

```diff
@@ -120,15 +117,14 @@ describe('Plugin transfers (optimistic)', function () {
       this.pluginA.send(Object.assign({
         id: id,
         amount: '1.0',
-        data: new Buffer(''),
-        noteToSelf: new Buffer(''),
+        data: '',
+        noteToSelf: '',
         executionCondition: undefined,
-        cancellationCondition: undefined,
         expiresAt: undefined
       }, transferA)).catch(done)
     })
 
-    it('should reject optimistic transfer with repeat id', function (done) {
+    it('should not reject optimistic transfer with repeat id', function (done) {
```

Comment:
I'll change the wording on that, but @justmoon was saying that a repeat transfer should not be submitted again, but it also should not create an error (because the `send` call has technically completed its job by sending the transfer at a previous time)